packages/dx-chart-core/src/plugins/series/computeds.test.js

```diff
@@ -343,3 +344,9 @@ describe('checkZeroStart', () => {
     expect(fromZero).toEqual({ axis1: false });
   });
 });
+
+describe('get items', () => {
```

Comment:
Usually we use for describing message the following text `#getPieItems`. [Link](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-grid-core/src/plugins/custom-grouping/computeds.test.js#L19)